reactioncommerce/reaction
# Multiple Refunds Not Showing

**Resolves**
#2285 

**Testing**
1. Place and Approve/Capture an order using Paypal Express
1. Issue several refunds
1. Observe that they are reflected.
